static/scss/partials/glocal-menu.scss

```diff
@@ -17,16 +17,16 @@ glocal-menu {
   position: absolute;
   visibility: hidden;
   color: #000000;
-  right: 28px;
+  right: $spacing-lg;
   border-radius: 8px;
   box-shadow: 0 7px 12px -3px rgba(28, 28, 29, .502);
   width: auto;
-  max-width: $screen-sm;
+  max-width: calc(#{$screen-sm} - #{$spacing-lg});
```

Comment:
Calculating a smaller width based on Protocol tokens so that the width of the component does not overflow.